chapters/synchronous.tex

```diff
@@ -6,7 +6,7 @@ \chapter{Synchronous Language Elements}\label{synchronous-language-elements}
 variables and equations, as well as an additional kind of \lstinline!when!-clause.
 The benefits of synchronous behavior is that it allows a model to define large
 sampled data systems in a safe way, so that the
-translator can provide good diagnostics in case of a modeling error.
+translator can provide good a diagnostic in case of a modeling error.
```

Comment:
Same comment about updating to one sentence per line, since we are changing it anyway.

Reply:
Ok